src/engine/ruleset/decoders/windows/windows_powershell.yml

```diff
@@ -0,0 +1,74 @@
+name: decoder/windows_powershell/0
+
+sources: 
+  - decoder/windows_event_decoder/0
```

Comment:
Please, change the underscores (`_`) to dashes (`-`).

Reply:
Done